--- origsrc/openbox-3.5.0/Makefile.am	2011-08-01 15:14:58.000000000 -0500
+++ src/openbox-3.5.0/Makefile.am	2011-09-07 23:44:11.684646500 -0500
@@ -77,8 +77,10 @@ obrender_libobrender_la_CPPFLAGS = \
 	-DG_LOG_DOMAIN=\"ObRender\" \
 	-DDEFAULT_THEME=\"$(theme)\"
 obrender_libobrender_la_LDFLAGS = \
+	-no-undefined \
 	-version-info $(RR_CURRENT):$(RR_REVISION):$(RR_AGE)
 obrender_libobrender_la_LIBADD = \
+	obt/libobt.la \
 	$(X_LIBS) \
 	$(PANGO_LIBS) \
 	$(GLIB_LIBS) \
@@ -122,6 +124,7 @@ obt_libobt_la_CPPFLAGS = \
 	-DDATADIR=\"$(datadir)\" \
 	-DCONFIGDIR=\"$(configdir)\"
 obt_libobt_la_LDFLAGS = \
+	-no-undefined \
 	-version-info $(OBT_CURRENT):$(OBT_REVISION):$(OBT_AGE)
 obt_libobt_la_LIBADD = \
 	$(XINERAMA_LIBS) \
